webapp/components/experiments/Experiments.js

```diff
@@ -154,80 +160,97 @@ const Experiments = (props) => {
     const date = new Date(timestamp);
     return date?.toLocaleDateString();
   };
+
+  const [dialogOpen, setDialogOpen] = useState(false)
+
+  const [list, setList] = useState(true);
+
+  const createExperimentScreen = () => {
+    setDialogOpen(false)
+    setList(false)
+  }
   return (
-    <GridLayout className={classes.root}>
-      <div>
-        <Box>
-          <TableContainer component={Paper}>
-            <Table aria-label="simple table">
-              <TableBody>
-                {experiments.map((experiment) => (
-                  <TableRow key={experiment?.name}>
-                    <TableCell component="th" scope="row">
-                      <Button>
-                        <Typography variant="h6" className="experimentHead">
-                          {experiment?.name}
+    <>
+    { list ?
+      <GridLayout className={classes.root}>
```

Comment:
I would suggest to extract the Experiment list to a separate component with its own state, the component that decides which view to show (list, detail, edit, ..) should not know of the experiment list state and its actions, ensures also a valid state for the Experiment list by design